enacts/config-test-enacts.yaml

```diff
@@ -67,6 +67,29 @@ maprooms:
               map_max: 40
 
     wat_bal: null
-    crop_suitability: null
+    crop_suitability:
+    - core_path: crops_climate_suitability
+      title: CSC
```

Comment:
I suggest that `core_path` and `title` should have reasonable defaults, so we don't have to specify them unless we want to customize them.

If you want to have multiple instances, you'll have to customize the path and title, but it seems like having only one may be the most comon arrangement.